yarn.lock

```diff
@@ -990,7 +990,7 @@ append-transform@^1.0.0:
   dependencies:
     default-require-extensions "^2.0.0"
 
-aproba@^1.0.3, aproba@^1.1.1:
+aproba@^1.1.1:
```

Comment:
This file should not have any changes in your PR, I think? /cc @kevinlee12

Reply:
I don't know why that changed, honestly. Should I revert it back to what it was?

Reply:
Not sure. I'll defer to @kevinlee12 to confirm.

Reply:
Apparently it was updated, I've made https://github.com/oppia/oppia/pull/8316 to address this. In the meantime, please revert it for this PR. thanks!

Reply:
I've reverted the changes with my latest commit. @kevinlee12 PTAL.